index.bs

```diff
@@ -1150,10 +1150,12 @@ run the following steps:
 1. Let |attributes| be a new [=/list=].
 1. If |domain| is not null, then run these steps:
     1. If |domain| starts with U+002E (`.`), then return failure.
-    1. If |host| does not equal |domain| and
-        |host| does not end with U+002E (`.`) followed by |domain|,
+    1. Let |parsedDomain| be the result of [=host parser|host parsing=] |domain|.
+    1. If |parsedDomain| is a failure, then return failure.
+    1. If |host| does not equal |parsedDomain| and
+        |host| does not end with U+002E (`.`) followed by |parsedDomain|,
```

Comment:
Use https://infra.spec.whatwg.org/#string-ends-with here?

Reply:
Thanks!

now n/a since we call into the other algo where this check is done